packages/components/tests/e2e/specs/vsCodeExtension.spec.js

```diff
@@ -747,6 +747,35 @@ context('VS Code Extension', () => {
       cy.get('.trace-node[data-event-id="15"]')
         .should('be.visible')
         .should('have.class', 'highlight');
+
+      cy.get('.trace-node[data-event-id="13"]')
+        .click()
+        .should('have.class', 'highlight');
+      cy.get('.trace-node[data-event-id="1"]').should(
+        'not.have.class',
+        'filtered'
+      );
+      cy.get('.trace-node[data-event-id="3"]').should(
+        'not.have.class',
+        'filtered'
+      );
+      cy.get('.trace-node[data-event-id="15"]').should(
+        'not.have.class',
+        'highlight'
+      );
+
+      cy.get('.details-panel__buttons').contains('Clear selection').click();
```

Comment:
I'd expect that `Clear selection` removes the active selection entirely, i.e. a highlighted event is not then selected

Reply:
That action on line 767 removes active selection from event id=13 (which is not present in filter input), but it was clicked by user on line 752.

Reply:
After that, previously selected event (15) became selected again and user can continue loop through filtered events.

Reply:
`Clear selection` should remove the active selection entirely. It's unexpected that it selects another event and following that the button then does nothing.

Reply:
Got it. Now `Clear selection` button will make empty Trace view filter input.
https://github.com/applandinc/appmap-js/pull/519/commits/b346b187ba21a6d25084edf2372d22168ea3a58c